docker/docker-compose.dev.yml

```diff
@@ -27,14 +28,14 @@ services:
       - COMMAND=create_initial_data
       - DATASTORE_INITIAL_DATA_FILE=https://raw.githubusercontent.com/OpenSlides/OpenSlides/openslides4-dev/docs/example-data.json
       - OPENSLIDES_DEVELOPMENT=1
-    ports:
```

Comment:
This breaks https://github.com/OpenSlides/OpenSlides/blob/openslides4-dev/dev-commands/set-ds.sh

Please add it back or implement a different solution to `set-ds.sh`